cmd/meroxa/turbine/utils.go

```diff
@@ -31,6 +31,13 @@ const (
 	IncompatibleTurbineVersion = `your Turbine library version is incompatible with the Meroxa CLI.
 For guidance on updating to the latest version, visit:
 https://docs.meroxa.com/beta-overview#updated-meroxa-cli-and-outdated-turbine-library`
+
+	grpcFuncCollectionErr = "invalid ProcessCollectionRequest.Collection: embedded message failed validation | " +
+		"caused by: invalid Collection.Name: value length must be at least 1 runes"
+	grpcDestCollectionErr = "invalid WriteCollectionRequest.SourceCollection: embedded message failed validation | " +
+		"caused by: invalid Collection.Name: value length must be at least 1 runes"
+	missingSourceCollectionErr = `missing source collection, please ensure that you have configured your source correctly:
```

Comment:
it could be a whole source or the source collection, right?
```suggestion
	missingSourceCollectionErr = `missing source or source collection, please ensure that you have configured your source correctly:
```

Reply:
yup - I'll commit this change. thanks!